microshift_install/microshift-install-rpm.adoc

```diff
@@ -5,19 +5,22 @@ include::_attributes/attributes-microshift.adoc[]
 :context: microshift-install-rpm
 toc::[]
 
-You can install MicroShift from an RPM package on a machine with {op-system-first} {op-system-version}. 
+You can install {product-title} from an RPM package on a machine with {op-system-first} {op-system-version}. 
 
-To embed MicroShift in a {op-system-ostree} image, see xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Embedding MicroShift into a {op-system-ostree-first} image].
+[IMPORTANT]
+====
+Red Hat will not provide or support an update or upgrade path from Developer Preview and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```

Comment:
```suggestion
Red Hat will not provide or support an update or upgrade path from the Developer and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```

Reply:
The original text is QE/SME approved so I don't think I'll change it.